content/en/blog/_posts/2023-MM-DD-kubelet-podresources-api-ga.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog
+title: 'Kubernetes 1.28: Node podresources API Graduates to GA'
+date: 2023-MM-DD
```

Comment:
Hi @ffromani , here Comm Shadow from 1.28 release, could you please edit for the scheduled publication date? 
```suggestion
date: 2023-08-23
```

Reply:
thanks, fixed